ringscout.gs

```diff
@@ -1,6 +1,13 @@
+"use strict";
+/******************************************************************************/
+// Configuration
+/******************************************************************************/
+
 // Grab config variables from the Google Sheet
 var ss = SpreadsheetApp.getActiveSpreadsheet();
 var sheet = ss.getActiveSheet();
+var USERNAME = sheet.getRange(21, 2).getValue();
```

Comment:
right now, these do nothing until I can learn how to pass a variable within another variable... something about constructors...